main.py

```diff
@@ -13,27 +14,55 @@ def tweet(message: str):
     auth.set_access_token(os.getenv("ACCESS_TOKEN"), os.getenv("ACCESS_TOKEN_SECRET"))
 
     api = tweepy.API(auth)
-    api.update_status(message)
+    try:
+        api.update_status(message)
+        logger.info("The gazette was posted on twitter!")
+    except tweepy.TweepError as e:
+        logger.exception(e)
+
+
+def create_maria_quiteria_api_token():
+    headers = {"content_Type": "application/json"}
+    url = f"{os.getenv('MARIA_QUITERIA_API_HOST')}/token/"
+    data = {
+        "username": f"{os.getenv('MARIA_QUITERIA_USERNAME_CREDENCIALS')}",
+        "password": f"{os.getenv('MARIA_QUITERIA_PASSWORD_CREDENCIALS')}",
+    }
+
+    token_response = requests.post(url, headers=headers, data=data)
+    logger.info("Getting token from Maria Quitéria")
+
+    if token_response.status_code == 200:
+        return f"Bearer {token_response.json()['access']}"
+    else:
+        logger.error(token_response.raise_for_status())
 
 
 def get_todays_gazette():
     gazettes = []
 
     date_today = datetime.date.today().strftime("%Y-%m-%d")
 
+    token_maria_quiteria = create_maria_quiteria_api_token()
+
     params = {"start_date": date_today}
-    url = os.getenv("MARIA_QUITERIA_API_URL")
+    url = f"{os.getenv('MARIA_QUITERIA_API_HOST')}/gazettes/"
     headers = {
         "Content-Type": "application/json",
-        "Authorization": os.getenv("MARIA_QUITERIA_API_TOKEN"),
+        "Authorization": token_maria_quiteria,
```

Comment:
```suggestion
        "Authorization": f"Bearer {token_maria_quiteria}",
```
Seguindo a sugestão anterior